app/store/index.ts

```diff
@@ -14,8 +15,12 @@ import { wallet } from './wallet'
 export const store = configureStore({
   reducer: {
     block: block.reducer,
-    wallet: wallet.reducer
-  }
+    wallet: wallet.reducer,
+    oceanInterface: oceanInterface.reducer
```

Comment:
```suggestion
    ocean: ocean.reducer
```

I think we can just call this `ocean`